src/index.js

```diff
@@ -555,15 +555,17 @@ class JsonSchemaForm extends React.Component {
         </View>
         {!!isSubmitError && <Text style={formStyles.error}>Please fill all required fields.</Text>}
         {children || (submitButton === false && cancelButton === false) ? children : (
-          <>
+          <View style={formStyles.buttonsBlock}>
             {cancelButton ? (
               <CancelButton
                 onPress={this.onCancel}
+                style={{marginRight:5}}
```

Comment:
please move that style in StyleSheet

Reply:
Fixed. Please see commit
https://github.com/velocitycareerlabs/react-native-jsonschema-form/pull/44/commits/178b462fc6429ef18dfef0901bf797611fc81ff6